python/lsst/ts/standardscripts/maintel/m2/check_actuators.py

```diff
@@ -0,0 +1,262 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["CheckActuators"]
+
+
+import asyncio
+import time
+
+import numpy as np
+import yaml
+from lsst.ts.idl.enums.Script import ScriptState
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+
+from ...base_block_script import BaseBlockScript
+
+# TODO: move constants from lsst.ts.m2com to ts-xml
+NUM_ACTUATOR, NUM_TANGENT_LINK = 78, 6
```

Comment:
I think it would be better to split these in 2 lines:

```
NUM_ACTUATOR = 78
NUM_TANGENT_LINK = 6
```

Makes it easier to see what's what.